### ISSUE TITLE
Lots of Markbot Issues

### ISSUE THREAD

**Issue by planetorium** [planetorium/animal-rescue-website#1]
@hellachella

Hi Chelle, 

I am having a lot of issues with my markbot at the moment. I'm not sure if it's just mine, or if it's other people as well, but it's still asking me for html files we don't need.

<img width="506" alt="screen shot 2017-12-28 at 5 50 15 pm" src="https://user-images.githubusercontent.com/22224512/34425187-a89acbce-ebf7-11e7-9942-c274dacf95a3.png">

I have been using the other files it asks for, the rescues.html and adopt.html and it has been working fine. 

I'm also not sure if I completely missed something or if I'm blanking on a lesson, but I don't remember learning any of these terms or learning how to use them? 
<img width="506" alt="screen shot 2017-12-28 at 5 51 37 pm" src="https://user-images.githubusercontent.com/22224512/34425204-d57afc90-ebf7-11e7-8785-6872cc76033f.png">

The other issue is that it's telling me the language is wrong on only one of my html files, but it is set up exactly the same as the rest of them. 
<img width="506" alt="screen shot 2017-12-28 at 5 52 59 pm" src="https://user-images.githubusercontent.com/22224512/34425217-04267222-ebf8-11e7-89b7-168085b0ceb8.png">

Every time I want to check my files and see how they're doing in markbot, the wheel never stops spinning and it takes at least 5 minutes to be able to see the results. I'm in the green for performance, so I'm not sure why it's taking so long. I also have to quit atom and markbot, and then reopen markbot for it to work again. 

Sorry for all the information and questions! I think my website is going relatively well so far, it's just the markbot issue that I'm having at the moment. If you have any feedback on what I have at the moment, let me know. 

Thanks and I hope you're having a great holiday, 

Victoria

**Comment by hellachella** [comment]
Did you by any chance move this file on your computer after cloning it?

**Comment by planetorium** [comment]
No, I didn't move it at all. I just cloned it onto my desktop like I usually do and it opened in github. I can try redownloading and then replacing all my files.

**Comment by hellachella** [comment]
@thomasjbradley
